src/index.js

```diff
@@ -1,28 +1,151 @@
+/* eslint-disable no-use-before-define */
 import './index.css';
 
-const tasks = [
-  { description: 'Work on a project', completed: true, index: 0 },
-  { description: 'Read a novel', completed: true, index: 1 },
-  { description: 'Learn Webpack', completed: false, index: 2 },
-  { description: 'Do House chores', completed: false, index: 3 },
-];
+let tasks = [];
 
 function displayTasks() {
   const todoList = document.getElementById('todo-list');
   todoList.innerHTML = '';
   tasks.sort((a, b) => a.index - b.index);
```

Comment:
- [ ] [REQUIRED] Great job so far @unorjikingsley , however, kindly note that a part of the project requirement states that the property index should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). You currently have the index of the first tasks as 0, you can add 1 to the index.
### Your Design
<img width="975" alt="Screenshot 2023-05-04 at 00 54 33" src="https://user-images.githubusercontent.com/58692590/236076254-89b9a82e-1d85-476a-914a-27c975b2d3d3.png">

Reply:
- [ ] Great job fixing the index numbering ✔️